README.rst

```diff
@@ -63,14 +63,69 @@ Publishing
     git tag 0.5.1
     git push origin 0.5.1
 
-4. Register the package::
+``.pypirc`` file
+-----------------
+
+To ensure publication and testing of publications are submitted to the correct
```

Comment:
hard to parse
does this mean
`To ensure publication and to test that publications are submitted to...`?

Or are we talking about submitting publications and testing of those publications?  I am confused.